WordPress/Classes/Services/EditorSettingsService.swift

```diff
@@ -1,11 +1,17 @@
 import Foundation
+import WordPressKit
 
 @objc enum EditorSettingsServiceError: Int, Swift.Error {
     case mobileEditorNotSet
 }
 
 @objc class EditorSettingsService: CoreDataService {
 
+    private lazy var coreDataStackSwift: CoreDataStackSwift = {
+        // The concrete type of coreDataStack is actually ContextManager, which also conforms to CoreDataStackSwift.
+        (coreDataStack as? CoreDataStackSwift) ?? ContextManager.shared
+    }()
```

Comment:
😫 Having to support both Swift and Objective-C interface is really annoying. Do you think we should use a computed property here rather than storing a duplicated reference to the same object?

Reply:
Yeah, sounds good to me. My intention was to avoid creating the context instance over and over, but since both the `coreDataStack` and the `shared` instance are already stored, this wouldn't happen.

I can create a quick PR for this bit tomorrow! Thanks.